operations/mimir-mixin/config.libsonnet

```diff
@@ -243,46 +261,46 @@
       baremetal: {
         // Somes queries does not makes sense when running mimir on baremetal
         // no need to define them
-        cpu_usage: 'sum by(%(instanceLabel)s) (rate(node_cpu_seconds_total{mode="user",%(namespace)s,%(instanceLabel)s=~".*%(instanceName)s.*"}[$__rate_interval]))',
+        cpu_usage: 'sum by(%(instanceLabel)s) (rate(node_cpu_seconds_total{mode="user",%(namespace)s,%(instanceLabel)s=~"%(instanceName)s"}[$__rate_interval]))',
         memory_working_usage:
           |||
-            node_memory_MemTotal_bytes{%(namespace)s,%(instanceLabel)s=~".*%(instanceName)s.*"}
```

Comment:
Note to reviewers: the `.*` prefix and suffix shouldn't be hardcoded in the query, but part of `instanceName`. For this reason, I moved it to `$._config.instance_names`.